### PR TITLE
Feature/drop databases script

### PR DESCRIPTION
**Pull request by iariasleon**
@frbattid @gtorodelvalle

### CODE REVIEW

**Comment by gtorodelvalle** [comment]
LGTM ;) Thanks @iariasleon ! ;)

File: resources/drop_database_mongo.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# You should have received a copy of the GNU Affero General Public License along with iot-sth. If not, see
+#    http://www.gnu.org/licenses/.
+#
+# For those usages not covered by the GNU Affero General Public License please contact:
+#    iot_support at tid.es
+#
+# author: 'Iv�n Arias Le�n (ivan.ariasleon at telefonica dot com)'
+#
+
+if [  "$1" == ""  ]
```
Comment (frbattid) [inline resources/drop_database_mongo.sh]: I would control the number of arguments passed to the script. If not correct, show the usage.

Comment (frbattid) [inline resources/drop_database_mongo.sh]: Anyway, this is Ok for me, NTC

File: test/perfomance/README.md [hunk 2]
```diff
@@ -26,8 +26,15 @@ nohup sh startAgent.sh --udp-port 0 --tcp-port 4444 &
 
   >**Scenario**:
 ```
-* N threads (notifications) in the same time against a STH instance specific, where threads are notifications total, entities are the number of entities in each service path
-and service path is the div between notifications (threads) and entities.
+* Multiples Notifications in the same time against a STH instance specific.
```
Comment (frbattid) [inline test/perfomance/README.md]: In the same time -> at the same time

Comment (iariasleon) [inline test/perfomance/README.md]: Fixed in 2e4a7bc

**Comment by frbattid** [comment]
LGTM

**Pull request merged**